lib/utils/hookFiles.js

```diff
@@ -0,0 +1,48 @@
+import pathModule from "path"
+import fs from "fs"
+
+const createHookFiles = ({ path, hookName, dumbString, indexString, options }) => {
+	const hooksPath = pathModule.resolve(path, "hooks")
+	const dumbhookPath = pathModule.resolve(hooksPath, `${hookName}.jsx`)
+	const styleSheetPath = pathModule.resolve(hooksPath, options.scss ? `${hookName}.scss` : `${hookName}.css`)
+	const indexPath = pathModule.resolve(path, "index.js")
+
+	fs.writeFile(indexPath, indexString, "utf8", (err) => {
+		if (err) {
+			throw err
+		}
+
+		console.log("Index created !")
+	})
+
+	fs.writeFile(dumbhookPath, dumbString, "utf8", (err) => {
+		if (err) {
+			throw err
+		}
+
+		console.log("Dumb hook created !")
+	})
+
+	fs.writeFile(styleSheetPath, "", "utf8", (err) => {
```

Comment:
Hello, does you hook have a stylesheet ? Is it a component ? I'm a bit lost here

Reply:
Yes, a hook returns a `React.element` or `Node.element`, and can therefore have a stylesheet.

Reply:
ok